config.go

```diff
@@ -66,6 +66,7 @@ const (
 	sampleConfigFilename         = "sample-btcd.conf"
 	defaultTxIndex               = false
 	defaultAddrIndex             = false
+	pruneMinSize                 = 1536
```

Comment:
Ah, I see there's another min size here. Shouldn't we sync this with what's being checked in `database/ffldb/db.go`?

Reply:
So the 1536 number came from the fact that we have these constants.

```
blocksNeeded = 288
ConsensusMaxBlockSize = 4,000,000
onDiskBlkFileSizeInBytes = 536,870,912
```

We need to keep 288 blocks to adhere to `NODE_NETWORK_LIMITED` BIP. Each block can be a maximum of 4MB. Each block file (XXXXXX.fdb) is 512MiB.

```
(blocksNeeded*ConsensusMaxBlockSize)/onDiskBlkFileSizeInBytes = 2.145
```

We do the math and we need to keep 2.14 on disk .fdb files to adhere to the `NODE_NETWORK_LIMITED` rule. Can't keep .14 files so we round up to 3. `512MiB * 3` is 1536MiB which is out `pruneMinSize`.

As mentioned [here](https://github.com/btcsuite/btcd/pull/1971#discussion_r1211140556), the check in `database/ffldb/db.go` is a different check so it's ok.

Reply:
Yeah I don't mind the intervals at all. It's hardly an issue now that the overall prune cache size isn't maxed at ~4G

Reply:
Ah okay, the two different limits now make sense to me, thanks for the explanation.